README.md

```diff
@@ -1 +1,12 @@
 # AnimalRhymes
+
+## Loose API Documentation
+GET /api/v1/rhymes?q={query}
```

Comment:
GET request URL path should be `/api/v1/savage-rhymes?q={query}`

Reply:
How about s/savage-rhymes/savageRhymes/gc